### ISSUE TITLE
Restore wallet with File Trust - Backup

### ISSUE THREAD

**Issue by mrduumiii** [NewHorizonLabs/TRX-Wallet#9]
Hello developer.
I'm having a problem about my Tron Wallet being used
I can't remember my Private Key code.
But I have a Trust Backup file. Can I restore my wallet ?.
  Please help me

**Comment by tasmannelson** [comment]
Yes help this person please.  I am in the same boat.  Not a good look we use a recommended wallet and it’s not working through no fault of our own.  Moral is low can anyone out there help rebooting a wallet via trust file.  Thanks for reading

**Comment by tasmannelson** [comment]
Hello.  I have installed trust wallet and copied my trust file and password and managed to upload my wallet.  I can’t see my frozen tokens yet but I have my wallet.  So it’s a start!
